src/devices/Vcnl4040/Vcnl4040Device.cs

```diff
@@ -0,0 +1,135 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Device.I2c;
+using System.IO;
+using Iot.Device.Vcnl4040.Definitions;
+using Iot.Device.Vcnl4040.Internal;
+
+namespace Iot.Device.Vcnl4040
+{
+    /// <summary>
+    /// Represents a VCNL4040 device
+    /// </summary>
+    public class Vcnl4040Device : IDisposable
+    {
+        /// <summary>
+        /// I2C bus address
+        /// </summary>
+        public static int DefaultI2cAddress = 0x60;
```

Comment:
As it's a default address and it's not supposed to change, why not using a const?

Reply:
Agreed.